docs: call out coder/skills setup skill on install and quickstart pages (#25194)
coder/coder: docs/tutorials/quickstart.md

- Familiarity with running commands in the terminal
- 10 minutes of your time

> [!TIP]

> > If you use a coding agent like Claude Code, the [coder/skills](https://github.com/coder/skills) `setup` skill can walk it through the steps below (install Docker, install Coder, create your first template, and launch a workspace).

Reviewer: ```suggestion
> If you use a coding agent like Claude Code, the [coder/skills](https://github.com/coder/skills) `setup` skill can train the coding agent on the following steps (install Docker, install Coder, create your first template, and launch a workspace).
```
